lib/rubocop/cop/rspec/factory_bot/create_list.rb

```diff
@@ -71,7 +71,7 @@ class CreateList < Base
             (send {nil? #factory_bot?} :create_list (sym _) (int $_) ...)
           PATTERN
 
-          def on_block(node)
+          def on_block(node) # rubocop:disable InternalAffairs/NumblockHandler
```

Comment:
create_list yields, so it could be used with a numblock. Though I guess the cop doesn't really support that. Perhaps we have to open an issue to add support

Reply:
Indeed it is. we will mark it as a rubocop:todo comment.